modules/getting-started/ref-audit-log-catalog-events.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+// assembly-audit-log.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-audit-log-catalog-events.adoc_{context}"]
+= Catalog events
+
+{product-short} audit logs can include the following catalog events:
+
+`CatalogEntityAncestryFetch`:: Tracks `GET` requests to the `/entities/by-name/:kind/:namespace/:name/ancestry` endpoint, which returns the ancestry of an entity
+`CatalogEntityBatchFetch`:: Tracks `POST` requests to the `/entities/by-refs` endpoint, which returns a batch of entities
+`CatalogEntityDeletion`:: Tracks `DELETE` requests to the `/entities/by-uid/:uid` endpoint, which deletes an entity
+
+[NOTE]
+====
+You can restore the deleted entity if the parent location is still present in the catalog.
+====
```

Comment:
So end users won't have a way to manually restore deleted entities. What was I intended to say was, that even if this entity is deleted, if the parent location exists, then it'll be added back into the catalog in the next processing cycle.

Reply:
Ah, got it. That's what using active voice does -- exposes who or what is actually performing an action. I'll rewrite this note to clarify that it isn't the end user doing the restoring.